templates/membership/register.html.twig

```diff
@@ -12,7 +12,7 @@
     {% import 'javascript.js.twig' as js %}
     <script type="text/javascript">
         Kernel.onLoad(function() {
-            App.runRegistration();
+            App.runRegistration('{{ form_type|replace({'\\': '\\\\'}) }}');
```

Comment:
`form_type|js`?

Reply:
this filter does not exist , are you sure ?

Reply:
He meant `e('js')`